dockstore-webservice/src/main/java/io/dockstore/webservice/resources/proposedGA4GH/ToolsApiExtendedServiceImpl.java

```diff
@@ -167,27 +169,29 @@ public Response organizationsGet(SecurityContext securityContext) {
     public Response toolsIndexGet(SecurityContext securityContext) {
         if (!config.getEsConfiguration().getHostname().isEmpty()) {
             List<Entry> published = getPublished();
```

Comment:
Some day this may be too big to fit in memory, but we can cross that bridge later. Also, it was also like this, but `getPublished()`, gets tools, gets workflows, combines them into one list, then in `bulkUpsert` we break the list apart into tools and workflows.